packages/jira-adapter/src/client/connection.ts

```diff
@@ -39,12 +44,28 @@ const getBaseUrl = async (
   return response.data.baseUrl
 }
 
+const isSandBox = async (connection: clientUtils.APIConnection): Promise<boolean> => {
+  const baseURL = await getBaseUrl(connection)
+  return baseURL.includes('-sandbox-')
+}
+
+const isProd = async (connection: clientUtils.APIConnection): Promise<boolean | undefined> => {
```

Comment:
I would add a comment here explaining that based on the current data we extract we cannot be sure that the account isProduction but sometimes we can be sure that it is no a Production account
```suggestion
const isProd = async (connection: clientUtils.APIConnection): Promise<false | undefined> => {
```